### DISCUSSION TITLE
Add device class for sensor gas

### DISCUSSION THREAD

**Discussion by frenck** [home-assistant/architecture#538] [Entity Models] [answered]
## Context

Gas consumption is measured in volume.

## Proposal

Add a device class `DEVICE_CLASS_GAS`, for measuring gas consumption.

Additionally add the following units:

```py
VOLUME_CM3 = "cm³"
VOLUME_M3 = "m³"
```

For the frontend it would use icon: `mdi:gas-cylinder`

This can be used by e.g., the Toon and DSMR integrations.

## Consequences

Needs to be implemented.

**Comment by MartinHjelmare** [comment, marked as answer]
This was solved here:
https://github.com/home-assistant/core/pull/54110
